FinSearchUnified/Bundle/SearchBundleFindologic/QueryBuilder.php

```diff
@@ -66,23 +65,15 @@ public function __construct(
     }
 
     /**
-     * @param bool $isSearch Defaults to true
-     *
      * @return string|null
      */
-    public function execute($isSearch = true)
+    public function execute()
     {
-        if ($isSearch) {
-            $endpoint = self::SEARCH_ENDPOINT;
-        } else {
-            $endpoint = self::NAVIGATION_ENDPOINT;
-        }
-
         $url = sprintf(
             '%s/%s/%s?%s',
             self::BASE_URL,
             $this->shopUrl,
-            $endpoint,
+            static::ENDPOINT,
```

Comment:
Are we sure that `self::ENDPOINT` won't work here?

Reply:
Yes it was getting the Querybuilder's `index` endpoint, not the child class's overridden endpoint.

Reply:
I checked the PHP manual (just because I was curious). It seems that constants are definitely not statics but you have to access it this way since the constant is shared between multiple classes (QueryBuilder and Navigation-/SearchQueryBuilder).

The more you know.